src/Microsoft.VisualStudio.SlnGen.Tool/Microsoft.VisualStudio.SlnGen.Tool.csproj

```diff
@@ -13,4 +13,7 @@
   <ItemGroup>
     <FilesToSign Include="$(IntermediateOutputPath)$(TargetName)$(TargetExt)" Authenticode="Microsoft400" StrongName="StrongName" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Looks like this way it cannot find the package in `ProgramArguments.Shared.cs` and fails the build :(

Reply:
Add the PackageReference to [Shared.props](https://github.com/microsoft/slngen/blob/master/src/Shared/Shared.props#L16) which is imported by both projects.  We ship a .NET Core global tool and a .NET Framework console app.

Reply:
OK, thanks for the pointer. I've updated accordingly.